ProjectSidewalk/SidewalkWebpage
# Reduce Admin Page Initial Load Time by ~10s (Seattle) by Moving Coverage Data to new API Route

Working towards #380 

I called that API route in Admin.js and loaded that data onto the Admin page after its initial load from the server. I also made it so that the spinner stops the loading animation only after that API call happens so we won't get any proxy errors, but everything will still be loaded once the spinner stops.

This reduces the initial load time of the page by ~10 seconds, but the total load time remains the same since that data is queried after the initial page load. I have the Seattle database, but this change should induce speedups in every city's Admin page.

##### Before/After screenshots (if applicable)

##### Testing instructions
1. Load the admin page and verify that everything looks good! Additionally, you can time the amount of time it takes 

##### Things to check before submitting the PR
